Pipfile

```diff
@@ -19,6 +19,8 @@ markdownify = "==0.4.1"
 stackexchange = "==0.5.0"
 tabulate = "==0.8.7"
 GPUtil = "==1.4.0"
+flake8 = "*"
```

Comment:
`flake8` is already included in `[dev-packages]` no need to re-include it in `[packages]`